src/common/import_session.c

```diff
@@ -246,8 +246,24 @@ const char *dt_import_session_name(struct dt_import_session_t *self)
   return self->vp->jobcode;
 }
 
+/* Extends the filename pattern and removes the trailing white spaces, if any.
+ * (Trailing whitespaces after the filename extension could be confusing
+ * when the type of a file is decided from the filename extension.)
+ *
+ * The returned string should be freed with g_free() when no longer needed.
+ */
+char *_import_session_filename_from_pattern(struct dt_import_session_t *self, gchar *pattern)
+{
+  gchar *result_fname = NULL;
+
+  result_fname = dt_variables_expand(self->vp, pattern, TRUE);
+  g_strchomp(result_fname);
```

Comment:
both lines below are equivalent to:
`return g_strchomp(result_fname);`

Reply:
Thank you for your comment. I have corrected the commit.